thredded.gemspec

```diff
@@ -43,24 +47,11 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec-rails'
   s.add_development_dependency 'rubocop', '0.32.0'
   s.add_development_dependency 'shoulda-matchers', '~> 2.7'
-  s.add_development_dependency 'sqlite3'
-  s.add_development_dependency 'timecop'
   s.add_development_dependency 'test-unit'
+  s.add_development_dependency 'timecop'
 
   # dummy app dependencies
-  s.add_development_dependency 'jquery-rails'
-  s.add_development_dependency 'pg'
   s.add_development_dependency 'mysql2'
+  s.add_development_dependency 'pg'
   s.add_development_dependency 'puma'
-
-  # debug dependencies
-  s.add_development_dependency 'byebug'
-  s.add_development_dependency 'flamegraph'
-  s.add_development_dependency 'pry-rails'
-  s.add_development_dependency 'rack-mini-profiler'
-  s.add_development_dependency 'derailed'
-
-  if RUBY_VERSION != '2.0.0'
-    s.add_development_dependency 'stackprof'
-  end
 end
```

Comment:
Holy crap these changes make the OCD in me all sorts of happy. :heart: :heart: :heart: :heart: :heart: